### PR TITLE
move bulk updates to sql layer

### PR DESCRIPTION
**Pull request by EloyMartinez**

### CODE REVIEW

**Review by gschoeni — CHANGES_REQUESTED** [review]
Can we add a unit test for inserting embeddings as well for future proofing?

File: src/server/src/controllers/workspaces/data_frames/columns.rs [hunk 1]
```diff
@@ -19,6 +19,7 @@ use serde_json::{json, Value};
 
 pub async fn create(req: HttpRequest, body: String) -> Result<HttpResponse, OxenHttpError> {
     let app_data = app_data(&req)?;
+    println!("bonjour bonjour");
```
Comment (gschoeni) [inline src/server/src/controllers/workspaces/data_frames/columns.rs]: Bonjour 👋

**Review by gschoeni — COMMENTED** [review]
Just one println! cleanup then it looks good!

File: src/server/src/controllers/workspaces/data_frames/columns.rs (hunk 1 above)
Comment (EloyMartinez) [reply src/server/src/controllers/workspaces/data_frames/columns.rs]: bonjour ça va

Comment (EloyMartinez) [reply src/server/src/controllers/workspaces/data_frames/columns.rs]: done!!!!

**Pull request merged**